gradle-plugin/kubernetes/src/main/java/org/eclipse/jkube/gradle/plugin/GradleUtil.java

```diff
@@ -82,11 +82,17 @@ public static JavaProject convertGradleProject(Project gradleProject) {
   }
 
   private static Properties extractProperties(Project gradleProject) {
-    return gradleProject.getProperties().entrySet().stream().filter(e -> Objects.nonNull(e.getValue()))
-        .reduce(new Properties(), (acc, e) -> {
-          acc.put(e.getKey(), e.getValue());
-          return acc;
-        }, (acc, e) -> acc);
+    Properties properties = new Properties();
+    Properties gradleProperties = gradleProject.getProperties().entrySet().stream().filter(e -> Objects.nonNull(e.getValue()))
+      .reduce(new Properties(), (acc, e) -> {
+        acc.put(e.getKey(), e.getValue());
+        return acc;
+      }, (acc, e) -> acc);
+    if (gradleProperties != null) {
+      properties.putAll(gradleProperties);
+    }
+    properties.putAll(System.getProperties());
+    return properties;
```

Comment:
Can be simplified to:
```suggestion
    return Stream.concat(gradleProject.getProperties().entrySet().stream(), System.getProperties().entrySet().stream())
        .filter(e -> Objects.nonNull(e.getValue()))
        .reduce(new Properties(), (acc, e) -> {
          acc.put(e.getKey(), e.getValue());
          return acc;
        }, (acc, e) -> acc);
```

We might also want to think about property precedence or priority - Should system properties override those provided by Gradle project?